.semaphore/publish.yml

```diff
@@ -0,0 +1,33 @@
+version: v1.0
+name: Publish Gem
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+blocks:
+  - name: Publish
+    run:
+      when: "branch = 'master'"
```

Comment:
Could you try pushing a lower  / test version and comment out these two lines to see if it's actually successfully pushing a package?

Reply:
Honestly, I don't think that will help as semaphore isn't even set up for this repo yet.

Reply:
oh it isn't even set up here? We may need to talk to Wnek or Jared to see how we can do that, then.

Reply:
Yeah, I'm sure that we don't have the privileges to add that.

Reply:
oh right, I forgot about that; yeah the semaphore build will have to be set up for this repo before we can even ship this PR, otherwise these scripts do nothing

I'll see if I can find time to do this at some point this week 😅 

ngl tho, not feeling super rushed on this since number inventory isn't the current priority for the team to get out the door

Reply:
I'm in no rush here either. Take your time.